scipy/stats/_covariance.py

```diff
@@ -379,13 +427,6 @@ def covariance(self):
         """
         return self._covariance
 
-    @property
-    def dimensionality(self):
-        """
-        Dimensionality of the vector space
-        """
-        return np.array(self._dimensionality, dtype=int)[()]
-
```

Comment:
Elimination of the `dimensionality` parameter is a separate thing. We can move it to another PR, but if it's easy enough to include here, note that it's all contained in the first commit, so it can be filtered out by reviewing the commits after the first separately.